client/app/bundles/course/admin/pages/CodaveriSettings/components/buttons/LiveFeedbackToggleButton.tsx

```diff
@@ -15,11 +16,17 @@ import CodaveriSettingsChip from '../CodaveriSettingsChip';
 interface LiveFeedbackToggleButtonProps {
   assessmentIds: number[];
   for: string;
-  hideChipIndicator?: boolean;
+  forSpecificAssessment?: boolean;
```

Comment:
booleans please follow `is` `has` prefix convention where possible instead of custom naming, unless absolutely necessary.  
`isSpecificAssessment` will work fine here.

Reply:
done